extensions/adobe/experience/offer-management/offer-activity.schema.json

```diff
@@ -0,0 +1,65 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/offer-activity",
+  "type": "object",
+  "title": "Activity",
+  "description": "Offer Activity",
+  "definitions": {
+    "offer-activity": {
+      "properties": {
+        "@id": {
+          "type": "string",
+          "format": "uri",
+          "title": "ID",
+          "description": "The unique identifier of the activity"
+        },
+        "xdm:name": {
+          "type": "string",
+          "title": "Name",
+          "description": "Activity name"
+        },
+        "xdm:status": {
+          "type": "string",
+          "title": "Status",
+          "description": "Activity Status",
+          "enum": ["draft", "pending", "live", "complete", "archived"],
+          "meta:enum": {
+            "draft": "Draft",
+            "pending": "Pending",
+            "live": "Live",
+            "complete": "Complete",
+            "archived": "Archived"
+          }
+        },
+        "xdm:startDate": {
+          "type": "string",
+          "title": "Start Date",
+          "description": "Activity Start Date"
+        },
+        "xdm:endDate": {
+          "type": "string",
+          "title": "End Date",
+          "description": "Activity End Date"
+        },
+        "xdm:placement": {
```

Comment:
what does this mean?  Are there a standard set of values/enums?

Reply:
Description is updated